src/include/platform/OTARequestorInterface.h

```diff
@@ -147,9 +147,34 @@ class OTARequestorInterface
     enum OTATriggerResult
     {
         kTriggerSuccessful = 0,
-        kNoProviderKnown   = 1
+        kNoProviderKnown   = 1,
+        kWrongState        = 2
     };
 
+    // Various actions to take when OnConnected callback is called
+    enum OnConnectedAction
```

Comment:
I don't think this needs to be in the interface file any more now that `ConnectToProvider` is a private function to OTARequestor class.

Reply:
Done